examples/aws-atlas-privatelink/Readme.md

```diff
@@ -0,0 +1,109 @@
+# Example - AWS and Atlas PrivateLink with Terraform
+
+This project aims to provide a very straight-forward example of setting up PrivateLink connection between AWS and MongoDB Atlas.
+
+
+## Dependencies
+
+* Terraform v0.12 or v0.13
```

Comment:
question, the provider works for terraform v0.12, but the example is for v0.13, given the versions.tf, would it not better to say the example was tested with v0.13? like in https://github.com/mongodb/terraform-provider-mongodbatlas/tree/master/examples/Atlas_Database_Users ?

Reply:
Updated Readme.md with correct Terraform version matching the versions.tf